imgops/README.md

```diff
@@ -2,6 +2,9 @@
 Local version of the imgops service
 
 ## Requirements
+* [Nginx with with image filter module](http://nginx.org/en/docs/http/ngx_http_image_filter_module.html)
+  * Linux: `sudo apt-get install nginx nginx-extras`
+  * Mac: `brew install nginx-full --with-image-filter`
```

Comment:
> Will new users have it installed?

No. You'd have to `brew tap homebrew/nginx-full` first then `brew install` or `brew install homebrew/nginx/nginx-full --with-image-filter` as you did.